frontend/shadow-cljs.edn

```diff
@@ -1,4 +1,4 @@
-{:source-paths ["src/cljs" "src/gen"]
+{:source-paths ["src/cljs" "src/gen" "src/leaflet_zoominfo"]
```

Comment:
I think I'd change this to src/thirdparty ("vendor" is also common)

Reply:
Cool, I'll make this change and then merge.